external/webserver/http_client.c

```diff
@@ -123,9 +123,11 @@ pthread_addr_t http_handle_client(pthread_addr_t arg)
 				if (p->keep_alive == 0) {
 					HTTP_LOGD("Client %d closing.\n", sock_fd);
 					close(p->client_fd);
+					break;
```

Comment:
Who will close the `msg_q`?

Reply:
after break at 128, mq_close will be called